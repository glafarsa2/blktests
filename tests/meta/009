#!/bin/bash
# SPDX-License-Identifier: GPL-3.0+
# Copyright (C) 2017 Omar Sandoval
#
# Test checking dmesg.

. tests/meta/rc

DESCRIPTION="check dmesg"

requires() {
	_have_writeable_kmsg
}

test() {
	echo "Running ${TEST_NAME}"
	fake_bug_on
	echo "Test complete"
}
