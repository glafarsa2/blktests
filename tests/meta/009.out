Running meta/009
Test complete
